tests/test_metadata.py

```diff
@@ -0,0 +1,70 @@
+"""Test Metadata Loading."""
+
+from os import environ
+from pathlib import Path
+
+from pytest import raises
+
+from sbot.metadata import METADATA_ENV_VAR, MetadataKeyError, load
+
+
+def test_metadata_env_var() -> None:
+    """Check that the env_var is as expected."""
+    assert METADATA_ENV_VAR == "SBOT_METADATA_PATH"
+
+
+def test_metadata_key_error() -> None:
+    """Test the key error exception."""
+    e = MetadataKeyError("beans")
+
+    assert str(e) == "Key \'beans\' not present in metadata, or no metadata was available"
+
+
+def test_load_no_env_var() -> None:
+    """Test the behaviour when the environment variable is not set."""
+    data = load()
+    assert data == {}
+    data = load(fallback={"beans": 5})
+    assert data == {"beans": 5}
+
+
+def test_load_file() -> None:
+    """Test that we can load a file."""
+    data_path = Path(__file__).parent
+    environ[METADATA_ENV_VAR] = str(data_path.absolute())
```

Comment:
Playing with the environment variables in tests is kinda ugly, and it relies on the assumption that this variable does not already exist in the environment - I think `test_load_no_env_var` will fail if it is set. I'm thinking we should decouple the environment variable interaction from the `load` function to make the latter easier to test.

Reply:
Tracked in #36.